pom.xml

```diff
@@ -1529,6 +1530,11 @@
                 </exclusions>
             </dependency>
             <dependency>
+                <groupId>com.googlecode.cqengine</groupId>
+                <artifactId>cqengine</artifactId>
+                <version>2.7.1</version>
```

Comment:
I'll enter a CQ for this tomorrow.  Shout if we have to have commons-collections4 and if the scalacheck needs to be more than a test dependency.  It is worth re-running ./build/calculate-cqs.sh.